### PR TITLE
Add `isRepoHome`

### PR DESCRIPTION
**Pull request by yakov116**
As per https://github.com/sindresorhus/refined-github/pull/3261#discussion_r443263751

### CODE REVIEW

**Comment by yakov116** [comment]
Hmm not sure how to fix it

	/sindresorhus/edit
  	/sindresorhus/notifications/

Should not match

**Comment by fregante** [comment]
They should match, they're not special URLs

**Comment by fregante** [comment]
https://github.com/fregante/notifications

**Comment by yakov116** [comment]
I should have said we don't want ittomatch

**Comment by fregante** [comment]
You can **move** most of those URLs to `isRepoHome` and then use this pattern:

https://github.com/fregante/github-url-detection/blob/aa530b93bf865f0203a3a23e79a2c26e71901c81/index.ts#L329-L330

e.g.

```js
collect.set('isRepoRoot', 
	...collect.get('isRepoHome') as string[], 
	'etc'
);
collect.set('isRepoTree', 
	...collect.get('isRepoRoot') as string[], 
	'etc'
);

**Comment by fregante** [comment]
> I should have said we don't want it to match

We do.

**Comment by fregante** [comment]
https://github.com/fregante/notifications is `repoHome`, `repoRoot` and `repoTree`

File: index.ts [hunk 1]
```diff
@@ -289,26 +289,26 @@ collect.set('isRepoIssueList', [
...
 	'https://github.com/sindresorhus/refined-github/',
+	'https://github.com/sindresorhus/notifications/',
+	'https://github.com/sindresorhus/edit',
+	'https://github.com/sindresorhus/search',
+	'https://github.com/sindresorhus/branches',
 ]);
 
 export const isRepoRoot = (url: URL | Location = location): boolean => /^(tree\/[^/]+)?$/.test(getRepoPath(url)!);
 collect.set('isRepoRoot', [
 	// Some tests are here only as "gotchas" for other tests that may misidentify their pages
```
Comment (fregante) [inline index.ts:304]: This note can be moved too

File: index.ts [hunk 2]
```diff
@@ -287,22 +287,28 @@ collect.set('isRepoIssueList', [
 	'https://github.com/sindresorhus/refined-github/issues?q=is%3Aclosed+sort%3Aupdated-desc',
 ]);
 
-export const isRepoRoot = (url: URL | Location = location): boolean => /^(tree\/[^/]+)?$/.test(getRepoPath(url)!);
-collect.set('isRepoRoot', [
+export const isRepoHome = (url: URL | Location = location): boolean => isRepoRoot(url) && getRepoPath(url) === '';
```
Comment (fregante) [inline index.ts:290]:
```suggestion
export const isRepoHome = (url: URL | Location = location): boolean => getRepoPath(url) === '';
```

Comment (yakov116) [inline index.ts:290]:
```suggestion
export const isRepoHome = (url: URL | Location = location): boolean => getRepoPath(url) === '';
```

Dont thing that is needed

**Comment by yakov116** [comment]
@fregante can you make a release?

**Pull request merged**